Example/Rudder-Appsflyer/_AppDelegate.m

```diff
@@ -18,6 +19,14 @@ @implementation _AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     // Override point for customization after application launch.
+    // First Initialize the Appsflyer iOS SDK
+    [[AppsFlyerLib shared] setAppsFlyerDevKey:@"<devKey>"];
+    [[AppsFlyerLib shared] setAppleAppID:@"<appleAppId"];
+    [AppsFlyerLib shared].isDebug = YES;
```

Comment:
Do we keep Debug flags on in production builds?

Reply:
@anujkaura , this is part of the example app, not a part of the device mode code